[Spike] Add proposal activities monitor (#57)
cennznet/app-gov: README.md

yarn gov-relayer:docker

# start webpack
yarn gov-relayer:webpack

# start the CLI
yarn relayer:run
```

## Deployment

### `gov-relayer`

- Ensure `apps/gov-relayer/package.json#dependencies` is up-to-date with the code base
- Run `yarn gov-relayer:build`

> - The folder `dist/apps/gov-relayer` are ready to be copy over to a container

Reviewer: ```suggestion
- The folder `dist/apps/gov-relayer` is ready to be copied over to a container
```
